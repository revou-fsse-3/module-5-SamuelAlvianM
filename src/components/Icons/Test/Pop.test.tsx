import React from 'react';
import { render } from '@testing-library/react';
import Pop from '../Pop';

test('Humidity component renders correctly', () => {
  const { container } = render(<Pop />);
  expect(container).toMatchSnapshot();
});

test('Visibility component has the correct link attribute', () => {
    const { container } = render(<Pop />);
    const svgElement = container.firstChild as HTMLElement;

    // Check the 'xmlns' attribute
    expect(svgElement.getAttribute('xmlns')).toBe('http://www.w3.org/2000/svg');
});