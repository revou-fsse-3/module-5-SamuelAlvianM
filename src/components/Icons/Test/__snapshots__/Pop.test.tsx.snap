// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Humidity component renders correctly 1`] = `
<div>
  <svg
    fill="none"
    height="12"
    viewBox="0 0 68 98"
    width="15"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M34 94C44 94 64 88.55 64 62.24C64 35.92 34 4 34 4C34 4 4 35.92 4 62.24C4 88.55 24 94 34 94Z"
      stroke="currentColor"
      stroke-linecap="round"
      stroke-linejoin="round"
      stroke-width="7"
    />
  </svg>
</div>
`;
